src/igvfd/schemas/prediction_set.json

```diff
@@ -259,6 +259,24 @@
         },
         "input_file_sets": {
             "description": "The file set(s) required for this prediction set."
+        },
+        "assessed_genes": {
+            "title": "Assessed Genes",
+            "description": "A list of genes assessed in this prediction set.",
```

Comment:
I think we need a more thorough description to distinguish assessed_genes from *_gene_list properties.

Reply:
Please explain the difference by directly referencing the other genes properties.